### ISSUE TITLE
When I try to visit to the Dokploy server it is giving an issue.

### ISSUE THREAD

**Issue by IcelabsSolutionsInc** [Dokploy/dokploy#703]
### To Reproduce

![image](https://github.com/user-attachments/assets/6d0ca3ff-3f79-4310-919d-dcc0567e97e0)

### Current vs. Expected behavior

I was expected to prompt username and password.

### Provide environment information

```bash
I am using latest version.
```

### Which area(s) are affected? (Select all that apply)

Application

**Comment by Siumauricio** [comment]
Go to your server and run this command 

docker service logs dokploy

Very probably you ran out of space

**Comment by IcelabsSolutionsInc** [comment]
> Go to your server and run this command
> 
> docker service logs dokploy
> 
> Very probably you ran out of space

Yeah, thank you I seen the issue  storage was completely used. I fixed using the below process.
1. I executed df -h so I found that my storage is full.
2. I executed docker system prune -a --volumes and cleaned the images of docker which was not used or was previous builds which free up 59% of storage.
3. I restarted the docker and after few moments it started working.

Thanks for your help.
